database/migrations/20210121_175511_user.go

```diff
@@ -0,0 +1,37 @@
+package main
+
+import (
+	"github.com/beego/beego/v2/client/orm/migration"
+)
+
+// User_20210121_175511 : DO NOT MODIFY
+type User_20210121_175511 struct {
+	migration.Migration
+}
+
+// DO NOT MODIFY
+func init() {
+	m := &User_20210121_175511{}
+	m.Created = "20210121_175511"
+
+	_ = migration.Register("User_20210121_175511", m)
+}
+
+// Up : Run the migrations
+func (m *User_20210121_175511) Up() {
+	m.SQL(`CREATE TABLE "user"
+		(
+			id SERIAL,
+			email varchar(128) NOT NULL,
+			encrypted_password varchar(128) NOT NULL,
```

Comment:
How about to make it `hashed_password`?

Reply:
updated https://github.com/Lahphim/go-crawler-challenge/pull/36/commits/6b4e9e74883bfbabaadfa773e7bc65798de3facf